tests/integration-tests/tests/schedulers/test_slurm.py

```diff
@@ -843,7 +843,7 @@ def _gpu_resource_check(slurm_commands, partition, instance_type, instance_type_
 def _test_slurm_version(remote_command_executor):
     logging.info("Testing Slurm Version")
     version = remote_command_executor.run_remote_command("sinfo -V").stdout
-    assert_that(version).is_equal_to("slurm 20.11.7")
+    assert_that(version).is_equal_to("slurm 20.11.8")
```

Comment:
not related to this patch, but I think this test should be moved as build or kitchen test

Reply:
created a SIM ticket for this